src/DatePicker/CalendarYear.js

```diff
@@ -67,27 +69,31 @@ class CalendarYear extends React.Component {
     container.scrollTop = scrollYOffset;
   }
 
-  handleTouchTap = (event, year) => {
-    if (this.props.onYearTouchTap) this.props.onYearTouchTap(event, year);
+  handleTouchTapYear = (event, year) => {
+    if (this.props.onTouchTapYear) this.props.onTouchTapYear(event, year);
   };
 
   render() {
     const years = this.getYears();
     const backgroundColor = this.context.muiTheme.datePicker.calendarYearBackgroundColor;
     const styles = {
-      position: 'relative',
-      height: 'inherit',
-      lineHeight: '36px',
-      textAlign: 'center',
-      padding: '8px 14px 0 14px',
       backgroundColor: backgroundColor,
+      height: 'inherit',
+      lineHeight: '35px',
       overflowX: 'hidden',
       overflowY: 'scroll',
+      position: 'relative',
+      textAlign: 'center',
+      display: 'flex',
+      flexDirection: 'column',
+      justifyContent: 'center',
     };
 
     return (
       <div style={styles}>
-        {years}
+        <div>
```

Comment:
I tried adding flexbox to just the external divs and arrange the years which are enhancedButtons using justifyContent etc, but since it encapsulates 100 years in it and each yearButton is a EnhancedButton. It worked perfectly in Firefox but breaks for chrome and safari.

Easier way was to encapsulate the years in a div and apply flex to it again so that if its 100 years in the year container or just three years all the years will be aligned centrally.